connectd/connectd.c

```diff
@@ -1272,42 +1272,6 @@ setup_listeners(const tal_t *ctx,
 		return NULL;
 	}
 
-	/* If we want websockets to match IPv4/v6, set it up now. */
-	if (daemon->websocket_port) {
```

Comment:
looks like this experiment of yours is redacted also from the BOLTs?

Reply:
It was: I withdrew the proposal.